messaging/usecase/student_usecase.go

```diff
@@ -171,8 +76,7 @@ func (s studentUseCase) ListenStudentEdit() {
 	<-forever
 }
 
-// ListenStudentDelete listens to the queue for any deleted students
-func (s studentUseCase) ListenStudentDelete() {
+func (s studentUseCase) RefactoredListenStudentCreation() (error, <-chan amqp.Delivery, chan bool) {
```

Comment:
please rename this method to smth more descriptive, especially describe the ACTION it performs.